fe/fe-core/src/main/java/org/apache/doris/planner/SortNode.java

```diff
@@ -68,17 +69,16 @@ public class SortNode extends PlanNode {
     /**
      * Constructor.
      */
-    public SortNode(PlanNodeId id, PlanNode input, SortInfo info, boolean useTopN,
-                    boolean isDefaultLimit, long offset) {
+    public SortNode(PlanNodeId id, PlanNode input, SortInfo info, boolean useTopN) {
```

Comment:
it is better that do not change this Constructor and add another one for nereids